### PR TITLE
feat(treesitter): add :InspectTree command

### PR DESCRIPTION
**Pull request by clason**

### CODE REVIEW

File: runtime/doc/treesitter.txt [hunk 1]
```diff
@@ -603,6 +603,8 @@ inspect_tree({opts})                           *vim.treesitter.inspect_tree()*
     to toggle the display of the source language of each node, and press
     <Enter> to jump to the node under the cursor in the source buffer.
 
+    Can also be shown with `:InspectTree`.
```
Comment (justinmk) [inline runtime/doc/treesitter.txt:606]:
```suggestion
    Can also be shown with *:InspectTree*.
```

**Review by justinmk — APPROVED** [review]
`:Inspect [arg]` (like `:checkhealth [arg]`) might still be worth considering later but this is fine too

**Comment by clason** [comment]
> `:Inspect [arg]` (like `:checkhealth [arg]`) might still be worth considering later but this is fine too

Yeah, but I think I'd like to wait for a few (at least one) more things to `:Inspect` to triangulate how this should behave (especially w.r.t. subcommands and completion).

**Comment by lewis6991** [comment]
Should we consider deprecating `show_tree()`? I can't imagine any application of using this in a script.

**Comment by clason** [comment]
It's already deprecated?

File: runtime/lua/vim/treesitter.lua [hunk 2]
```diff
@@ -424,6 +424,8 @@ end
 --- display of the source language of each node, and press <Enter> to jump to the node under the
 --- cursor in the source buffer.
 ---
+--- Can also be shown with `:InspectTree`. *InspectTree*
```
Comment (justinmk) [inline runtime/lua/vim/treesitter.lua:427]: ? ex command tags usually include `:`

Comment (clason) [reply runtime/lua/vim/treesitter.lua]: Yup, typo (I no longer see too well.)

**Pull request merged**